kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/IfcfgConfigWriter.java

```diff
@@ -79,12 +79,35 @@ public void visit(NetworkConfiguration config) throws KuraException {
 
         if (!netInterfaceConfigs.isEmpty()) {
             for (NetInterfaceConfig<? extends NetInterfaceAddressConfig> netInterfaceConfig : netInterfaceConfigs) {
-                writeConfig(netInterfaceConfig);
+                if (getInterfaceStatus(netInterfaceConfig) != NetInterfaceStatus.netIPv4StatusUnmanaged) {
+                    writeConfig(netInterfaceConfig);
+                }
                 writeKuraExtendedConfig(netInterfaceConfig);
             }
         }
     }
 
+    private NetInterfaceStatus getInterfaceStatus(
+            NetInterfaceConfig<? extends NetInterfaceAddressConfig> netInterfaceConfig) {
+        if (netInterfaceConfig == null) {
+            return NetInterfaceStatus.netIPv4StatusUnknown;
+        }
+        NetInterfaceStatus status = NetInterfaceStatus.netIPv4StatusUnknown;
+        for (NetInterfaceAddressConfig addresses : netInterfaceConfig.getNetInterfaceAddresses()) {
```

Comment:
Maybe this for cycle needs a review: it's a little bit too nested. W should try to simplify it, maybe defining some sub methods